rules/falco_rules.yaml

```diff
@@ -3238,3 +3238,10 @@
     command=%proc.cmdline pid=%proc.pid file=%fd.name parent=%proc.pname gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4] container_id=%container.id image=%container.image.repository)
   priority: WARNING
   tags: [filesystem, mitre_credential_access, mitre_discovery]
+
+- rule: PTRACE attached to process
+  desc: "This rule detects an attempt to inject code into a process using PTRACE."
+  condition: evt.type=ptrace and evt.dir=> and evt.arg.request=11 and proc_name_exists
```

Comment:
This is a really initiative!! Thanks!
 
Here are my two cents/questions:
1. Should we include `PTRACE_SEIZE` as well?
2. Why do we care `proc_name_exists`?
3. Security tooling might use ptrace capability (e.g. Falco in certain scenario), maybe we should add macro to exclude the known processes?

Reply:
Hey @Kaizhe , thank you for your feedback!
1. Sure, PTRACE_SEIZE makes sense to me, I added it in the condition
2. We saw some false positives without that macro
3. Sure, I added the macro (as an empty list)